collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -248,7 +248,7 @@ public void onCreate() {
                 mgr.getSingularProperty(PropertyManager.PROPMGR_DEVICE_ID));
 
         AuthDialogUtility.setWebCredentialsFromPreferences(this);
-        if(timber.log.BuildConfig.DEBUG) {
+        if(BuildConfig.DEBUG) {
```

Comment:
I believe there should be a space after `if` here. Might as well fix it since you're changing it.